src/mds/MDCache.h

```diff
@@ -568,6 +592,9 @@ class MDCache {
   friend class Locker;
   friend class Migrator;
   friend class MDBalancer;
+  // FIXME hack so that the StrayManager can re-insert inode to inode_map
+  // on abort_queue
+  friend class StrayManager;
```

Comment:
Do you have a plan for fixing this or are you just marking it for future reference?